# config.py

errCode = {
    0: '정상처리',
    -10: '실패',
    -100: '사용자 정보교환 실패',
    -101: '서버 접속 실패',
    -102: '버전처리 실패',
    -103: '개인방화벽 실패',
    -104: '메모리 보호실패',
    -105: '함수입력값 오류',
    -106: '통신연결 종료',
    -200: '시세조회 과부하',
    -201: '전문작성 초기화 실패',
    -202: '전문작성 입력값 오류',
    -203: '데이터 없음',
    -204: '조회가능한 종목수 초과',
    -205: '데이터 수신 실패',
    -206: '조회가능한 FID수 초과',
    -207: '실시간 해제 오류',
    -300: '주문 과부하',
    -301: '주문전문 작성 오류',
    -302: '주문전송 실패',
    -303: '주문수량 오류',
    -304: '주문가격 오류',
    -305: '주문실패',
    -306: '주문가격의 범위 초과',
    -307: '주문 수량 초과',
    -308: '주문자체 오류',
    -309: '주문차트 오류'
}

def errors(err_code):
    return errCode.get(err_code, "알 수 없는 오류")
